modules/installation-launching-installer.adoc

```diff
@@ -60,7 +60,7 @@ You can install {product-title} on a compatible cloud.
 
 [IMPORTANT]
 ====
-You can run the installation program only once, during initial installation.
+You can run the `create cluster` option of the installation program only once, during initial installation.
```

Comment:
nit: "option" -> "subcommand"?  Or just "command"?  Or "the installation program's `create cluster` only once"?  Or something ;).